spec/zoneddatetime.html

```diff
@@ -1406,40 +1401,39 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-nanosecondstodays" type="abstract operation">
+    <emu-clause id="sec-temporal-normalizedtimedurationtodays" type="abstract operation">
       <h1>
-        NanosecondsToDays (
-          _nanoseconds_: an integer,
+        NormalizedTimeDurationToDays (
+          _norm_: a Normalized Time Duration Record,
           _zonedRelativeTo_: a Temporal.ZonedDateTime,
           _timeZoneRec_: a Time Zone Methods Record,
           optional _precalculatedPlainDateTime_: a Temporal.PlainDateTime,
-        ): either a normal completion containing a Record with fields [[Days]] (an integer), [[Nanoseconds]] (an integer), and [[DayLength]] (an integer), or an abrupt completion
+        ): either a normal completion containing a Record with fields [[Days]] (an integer), [[Remainder]] (a Normalized Time Duration Record), and [[DayLength]] (an integer), or an abrupt completion
       </h1>
       <dl class="header">
         <dt>description</dt>
         <dd>
-          It converts a number of _nanoseconds_ relative to a Temporal.ZonedDateTime _zonedRelativeTo_ (if supplied), and converts it into a number of days and remainder of nanoseconds, taking into account any offset changes in the time zone of _zonedRelativeTo_.
+          It converts a normalized time duration _norm_ relative to a Temporal.ZonedDateTime _zonedRelativeTo_, and converts it into a number of days and a remainder normalized time duration, taking into account any offset changes in the time zone of _zonedRelativeTo_.
           It also returns the length of the last day in nanoseconds, for rounding purposes.
         </dd>
       </dl>
       <p>Unless _nanoseconds_ = 0, _timeZoneRec_ must have looked up both `getOffsetNanosecondsFor` and `getPossibleInstantsFor`.</p>
       <emu-alg>
-        1. If _nanoseconds_ = 0, then
-          1. Return the Record { [[Days]]: 0, [[Nanoseconds]]: 0, [[DayLength]]: nsPerDay }.
-        1. If _nanoseconds_ &lt; 0, let _sign_ be -1; else, let _sign_ be 1.
-        1. Let _startNs_ be ℝ(_zonedRelativeTo_.[[Nanoseconds]]).
-        1. Let _startInstant_ be ! CreateTemporalInstant(ℤ(_startNs_)).
-        1. Let _endNs_ be _startNs_ + _nanoseconds_.
-        1. If IsValidEpochNanoseconds(ℤ(_endNs_)) is *false*, throw a *RangeError* exception.
+        1. Let _sign_ be NormalizedTimeDurationSign(_norm_).
+        1. If _sign_ = 0, then
+          1. Return the Record { [[Days]]: 0, [[Remainder]]: _norm_, [[DayLength]]: nsPerDay }.
+        1. Let _startNs_ be _zonedRelativeTo_.[[Nanoseconds]].
+        1. Let _startInstant_ be ! CreateTemporalInstant(_startNs_).
+        1. Let _endNs_ be AddNormalizedTimeDurationToEpochNanoseconds(_norm_, _startNs_).
+        1. If ! IsValidEpochNanoseconds(_endNs_) is *false*, throw a *RangeError* exception.
```

Comment:
```suggestion
        1. Let _endNs_ be ? AddInstant(_norm_, _startNs_).
```

Can be shortened to `AddInstant`.